tekton/tekton_suite_test.go

```diff
@@ -0,0 +1,109 @@
+/*
+Copyright 2022 Red Hat Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package tekton
+
+import (
+	"context"
+	"go/build"
+	"path/filepath"
+	"testing"
+
+	ctrl "sigs.k8s.io/controller-runtime"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	clientsetscheme "k8s.io/client-go/kubernetes/scheme"
+
+	// "k8s.io/apimachinery/pkg/runtime"
+	appstudiov1alpha1 "github.com/redhat-appstudio/release-service/api/v1alpha1"
+
+	tektonv1beta1 "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+
+	"k8s.io/client-go/rest"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/envtest"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+	//+kubebuilder:scaffold:imports
+)
+
+var (
+	cfg       *rest.Config
+	k8sClient client.Client
+	testEnv   *envtest.Environment
+	ctx       context.Context
+	cancel    context.CancelFunc
+)
+
+func TestPipelineRun(t *testing.T) {
```

Comment:
This file is confusing to me. Where is the test? I assume it is this function, but I don't see it called anywhere. It looks like this just has a before and after with no actual test

Reply:
According to the ginkgo doc: 
```
Next we define a single testing test: func TestBooks(t *testing.T). This is the entry point for Ginkgo - the go test runner will run this function when you run go test or ginkgo.
```
It is implicitly called. I need to read a bit more to understand it properly, but it seems it initiates `t *testing.T` and then all the other functions might use it through its pointer as it is not called anywhere else. It would be much clear if all the other functions were explicitly called inside it.

Reply:
Ah, I've never used ginkgo, cool. I am not able to review this file as closely, but nothing *looks* bad in it to me